packages/components/src/Navbar/Component.purs

```diff
@@ -56,48 +55,47 @@ fromJSProps jsProps =
         "JUNIOR" -> JUNIOR
         _        -> KSF
 
-type State =
-  { collapsedNavVisibility :: Visibility  }
-
-initialState :: State
-initialState = { collapsedNavVisibility: Hidden }
-
-jsComponent :: React.ReactComponent JSProps
```

Comment:
You could remove `JSProps` and `fromJSProps` as well, they aren't going to be used with this code anymore.